docs/writing-rules/rule-syntax.md

```diff
@@ -187,6 +187,29 @@ Comparison expressions support simple arithmetic as well as composition with [bo
 
 Building off of the previous example this will still catch code like `set_port(80)` but will no longer catch `set_port(443)` or `set_port(8080)`.
```

Comment:
- Avoid rarely use phrasal verbs (generally avoid all phrasal verbs and idioms unless there is no better word) - https://english.stackexchange.com/questions/102057/what-is-the-meaning-of-build-off-of and see phrasal verbs in this section: https://developers.google.com/style/translation?hl=en#clear-language
- Avoid "will" unless you are talking about a specific time in the future, use present tense for most of the docs. See https://developers.google.com/style/tense?hl=en for more information.
- When using words such as "this" it is better to specify what we mean by it. I guess that here "this" means the rule (looking at the example given above).

- Optional - feel free to remove my suggestions in brackets (`(it is not divisible by two)` and `(the number is larger than 1024)` if you find them misleading. Stating the obvious might help.

```suggestion
Building on the previous example, this rule pattern matches code such as `set_port(80)` but does not match `set_port(443)` (it is not divisible by two) or `set_port(8080)` (the number is larger than 1024).
```

Reply:
Again, I did not touch this and I did not even write this. You should open a separate PR for this.